todo-list/src/App.js

```diff
@@ -1,13 +1,12 @@
-import { useState } from "react";
-import TodoItem from "./TodoItem";
+import * as React from "react";
+import WeeklyTodoList from "./WeeklyTodoList";
 import "./styles.css";
 import { ACCENT_COLOURS, COLOURS } from "./colours";
 
 function App() {
-  const numTodoItems = 10;
-  const [isDarkMode, setDarkMode] = useState(false);
-  const [accentColour, setAccentColour] = useState(ACCENT_COLOURS.Blue);
-  const [selectedDroplet, setSelectedDroplet] = useState(0);
+  const [isDarkMode, setDarkMode] = React.useState(false);
+  const [accentColour, setAccentColour] = React.useState(ACCENT_COLOURS.Blue);
```

Comment:
Why is it `React.useState()` instead of `useState()` here?

Reply:
Same thing. It is just the way I imported react. If you want you can change it back for consistency.